src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -64,4 +70,15 @@ void SamplingWithFrameTrackWidget::OnMultiplicityCorrectionCheckBoxClicked(bool
   ui_->significance_level_label_->setText(text);
 }
 
+void SamplingWithFrameTrackWidget::OnUpdateButtonClicked() {
```

Comment:
Does it make sense to add a test for this? You would probably need to make the Validator Mockable and inject it. I dont know if worth the effort